tubesync/healthcheck.py

```diff
@@ -12,24 +12,38 @@
 
 import sys
 import requests
-
+import os
 
 TIMEOUT = 5  # Seconds
 
-
-def do_heatlhcheck(url):
+def do_heatlhcheck(url, username, password):
     headers = {'User-Agent': 'healthcheck'}
-    response = requests.get(url, headers=headers, timeout=TIMEOUT)
+    if username and password:
+        response = requests.get(url, headers=headers, timeout=TIMEOUT, auth=(username, password))
+    else:
+        response = requests.get(url, headers=headers, timeout=TIMEOUT)
     return response.status_code == 200
 
+def get_http_auth():
+    if "HTTP_USER" in os.environ:
```

Comment:
```suggestion
    if "HTTP_USER" in os.environ and "HTTP_PASS" in os.environ:
```

Reply:
Adding the PASS as well, just in case someone only sets one or the other.
Healthcheck will be unsuccessful, but at least python will not crash :)